pom.xml

```diff
@@ -150,10 +150,10 @@
     <parquet.version>1.11.0</parquet.version>
     <protobuf.version>3.12.4</protobuf.version>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <ratis.version>2.0.0</ratis.version>
+    <ratis.version>2.2.0</ratis.version>
     <slf4j.version>1.7.30</slf4j.version>
     <jackson.version>2.11.1</jackson.version>
-    <ozone.version>1.1.0</ozone.version>
+    <ozone.version>1.2.1</ozone.version>
```

Comment:
remove this and the ozone related dependencies of dependencyManagement within this file.